### ISSUE TITLE
Using Tags Instead of Stable Branch

### ISSUE THREAD

**Issue by robelgeda** [spacetelescope/wfirst-tools#35]
Right now we use the sable branch for mybinder builds. I noticed that you can use git tags to specify the commits (the history) you would like to build in mybinder. Adding a tag on GitHub is like doing a versioned release of the code. This is a better way of keeping track of changes as well as keeping mybinder builds at sable points.

**Comment by k-gilbert** [comment]
This sounds like it would be a solid improvement in workflow to me.  @ariedel, do you agree?

**Comment by ariedel** [comment]
Yes, this seems like an excellent way to keep stable versions around.

**Comment by robelgeda** [comment]
Sounds like a plan!
